.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true,
+  },
+  extends: ['eslint:recommended', 'prettier'],
+  ignorePatterns: ['dist'],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+    allowImportExportEverywhere: true,
+  },
+  rules: {
+    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-undef': 'off',
+    'no-unused-vars': 'off', // prefer @typescript-eslint version
```

Comment:
Might be missing something but is the `typescript-eslint` version of this rule configured?

Reply:
It seems to be I still get the unused variable warning with this off and this config was copied from our ui repo as well